guides/release/tutorial/01-orientation.md

```diff
@@ -47,9 +47,21 @@ installing octane-app-blueprint
   create testem.js
   create tests/index.html
   create tests/test-helper.js
+
+Installing packages... This might take a couple of minutes.
 npm: Installing dependencies ...
 npm: Installed dependencies
-Successfully initialized git.
+
+Initializing git repository.
+Git: successfully initialized.
+
+Successfully created project super-rentals.
+We suggest you that you get started by typing:
+
+  $ cd super-rentals
+  $ yarn start
```

Comment:
😭

Reply:
Wait, this seems like a bug in CLI itself: https://github.com/ember-cli/ember-cli/blob/master/lib/commands/init.js#L119